client/components/Feature/AddFeatureMutation.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import Relay, { RelayMutationConfig } from 'react-relay';
+import RelayGraphQLMutation from '../../../node_modules/react-relay/lib/RelayGraphQLMutation';
+
+class AddFeatureMutation {
+  static serverQuery =
+    Relay.QL`mutation {
+        addFeature {
+            featureEdge {
+                __typename
+                node {
+                    name
+                    id
+                    description
+                    url
+                }
+            }
+        }
+    }`;
+  static optimisticQuery =
+    Relay.QL`mutation {
+        addFeature {
+            featureEdge {
+                node {
+                    name
+                    description
+                    url
+                }
+            }
+        }
+    }`;
+  config: ?Array<RelayMutationConfig>;
```

Comment:
@Neitsch how's this `?Array<RelayMutationConfig>` thing work? I don't think I've never seen it before.